project_custom_info/demo/custom.info.template.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model,model_id:id
+tpl_addressed,Addressed to...,res.partner,base.model_res_partner
```

Comment:
This is not correct, as the model should be project. And the idea is to put some data related to project, not PEGI.